src/store/getters.ts

```diff
@@ -68,7 +68,9 @@ export default {
     return state.currentFilingStatus
   },
 
-  // get last Change of Directors filing from list of past filings
+  /** Returns date of last Change of Directors filing from list of past filings. */
+  // FUTURE: this will break if we only retrieve a page of filings
```

Comment:
Re: Thor has been talking about changing the filings API (that we use for the history list) to be more efficient, including supporting paging.